src/kmod.rs

```diff
@@ -0,0 +1,108 @@
+//! Load and unload kernel modules.
+
+use libc;
+use std::ffi::CStr;
+use std::os::unix::io::AsRawFd;
+
+use errno::Errno;
+use Result;
+
+/// Loads an ELF image into kernel space, performs any necessary symbol relocations,
+/// initializes module parameters to values provided by the caller, and then runs the module's init function.
+/// This function requires privilege.
```

Comment:
What privilege? Probably root privileges is correct here, yes? Also, can you separate this into a separate paragraph by putting a blank line above it or integrate it into the existing one?